### PR TITLE
[SW-347] Remove unnecessary scrolling and typography changes

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/pages/CurrencySettings.vue [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
-    >
-      <div
-        class="row"
-        @click="switchCurrency(code)"
+    <div class="options">
+      <RadioButton
+        v-for="{ code, name, symbol }, index in currencies"
+        :key="`${index}-${code}`"
+        :value="current && current.currency == code"
+        :disabled="false"
```
Comment (konradodo) [inline src/popup/pages/CurrencySettings.vue:12]: :disabled="false" normaly is always false

File: tsconfig.json [hunk 2]
```diff
@@ -7,18 +7,20 @@
     "module": "ES6",
     "moduleResolution": "node",
     "paths": {
-      "@/*": ["src/*"]
+      // "@/*": ["src/*"]
```
Comment (konradodo) [inline tsconfig.json:10]: if not necessary, delete. if necessary add a comment why it is commented out

**Review by konradodo — APPROVED** [review]

File: src/popup/pages/CurrencySettings.vue [hunk 3]
```diff
@@ -1,34 +1,36 @@
...
+        :class="['currency', {active: current && current.currency == code}]"
```
Comment (pstachel) [inline src/popup/pages/CurrencySettings.vue:13]:
```suggestion
        :class="['currency', { active: current?.currency === code }]"
```

**Review by pstachel — CHANGES_REQUESTED** [review]

File: src/popup/pages/CurrencySettings.vue (hunk 3 above)
Comment (peronczyk) [reply src/popup/pages/CurrencySettings.vue:13]: No way of using `?.` within the template.

File: src/popup/pages/CurrencySettings.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/pages/CurrencySettings.vue:12]: Done

File: tsconfig.json (hunk 2 above)
Comment (peronczyk) [reply tsconfig.json:10]: I'll add additional comment on this.

**Review by pstachel — APPROVED** [review]

**Pull request merged**